FirebaseStorageSwift/Tests/Integration/StorageIntegration.swift

```diff
@@ -256,6 +270,27 @@ class StorageIntegration: XCTestCase {
     waitForExpectations()
   }
 
+  func testAttemptToUploadDirectory() throws {
+    let expectation = self.expectation(description: #function)
+
+    let fileName = "Home Improvement.numbers"
+    let bundle = Bundle(for: StorageIntegration.self)
+    let fileURL = try XCTUnwrap(bundle.url(forResource: fileName, withExtension: ""),
+                                 "Failed to get filePath")
+    let ref = storage.reference(withPath: "ios/public/" + fileName)
+    ref.putFile(from: fileURL) { result in
+      switch result {
```

Comment:
I am not a Swift export, but is there a more succinct way to verify that the task has failed? If might also be possible to validate the error message, which would make this test very much self-explanatory.

Reply:
True, I'll work on it!